Dry schema validation error for new fields `agg_data_provider_country` & `agg_provider_country`

Attempting a local transformation:

```
docker run --rm -e SKIP_FETCH_CONFIG=true \
                -e SKIP_FETCH_DATA=true \
                -v $(pwd)/.:/opt/traject \
                -v $(pwd)/../dlme-traject:/opt/traject/config \
                -v $(pwd)/../dlme-metadata:/opt/traject/data \
                -v $(pwd)/output:/opt/traject/output \
                suldlss/dlme-transform:latest \
                stanford/maps/data/kj751hs0595.mods
```
results in a dry schema validation error:
```
[ERROR] Transform produced invalid data.

The errors are: [#<Dry::Schema::Message text="must be a string" path=[:agg_data_provider_country] predicate=:str? input=["United States of America", "United States of America"]>, #<Dry::Schema::Message text="must be a string" path=[:agg_provider_country] predicate=:str? input=["United States of America", "United States of America"]>]}
```

--- a/lib/adjust_cardinality.rb
+++ b/lib/adjust_cardinality.rb
@@ -21,7 +21,8 @@ class AdjustCardinality
   end
 
   def flatten_top_level(attributes)
-    flatten = %w[id agg_data_provider agg_data_provider_country agg_provider agg_provider_country agg_is_shown_at agg_is_shown_by agg_preview]
+    flatten = %w[id agg_data_provider agg_data_provider_country agg_provider agg_provider_country
+                 agg_is_shown_at agg_is_shown_by agg_preview]
     attributes.except(*flatten).tap do |output|
       flatten.each do |field|
         next unless attributes.key?(field)

--- a/lib/adjust_cardinality.rb
+++ b/lib/adjust_cardinality.rb
@@ -21,7 +21,7 @@ class AdjustCardinality
   end
 
   def flatten_top_level(attributes)
-    flatten = %w[id agg_data_provider agg_provider agg_is_shown_at agg_is_shown_by agg_preview]
+    flatten = %w[id agg_data_provider agg_data_provider_country agg_provider agg_provider_country agg_is_shown_at agg_is_shown_by agg_preview]
     attributes.except(*flatten).tap do |output|
       flatten.each do |field|
         next unless attributes.key?(field)

